.npmrc

```diff
@@ -0,0 +1,2 @@
+//.npmrc file
+engine-strict = true
```

Comment:
What was the reasoning behind this? Was something not working with yarn?

Reply:
Not sure but I figured if this was an npm project, why not restrict it to only use npm? Is this bad form? yarn and npm are quite different imo.

Reply:
From the [docs](https://docs.npmjs.com/cli/v8/using-npm/config#engine-strict):

> engine-strict
> 
>     Default: false
>     Type: Boolean
> 
> If set to true, then npm will stubbornly refuse to install (or even consider installing) any package that claims to not be compatible with the current Node.js version.
> 
> This can be overridden by setting the --force flag.
>